ale_linters/php/php.vim

```diff
@@ -18,9 +19,9 @@ function! ale_linters#php#php#Handle(buffer, lines) abort
         " vcol is needed to indicate that the column is a character.
         call add(l:output, {
         \   'bufnr': a:buffer,
-        \   'lnum': l:match[2] + 0,
+        \   'lnum': l:match[3] + 0,
         \   'vcol': 0,
-        \   'col': 1,
+        \   'col': match(getline(l:match[3]), l:match[2]) + 1,
```

Comment:
Hmm, I wonder, could some characters from the string being used here be interpreted as special regular expression characters instead? It might be best to prefix the string with a `nomagic` flag. Maybe `'\V'`. See `:help /magic`